tests/test_nginx/test_nginx_timings_parser.py

```diff
@@ -6,6 +6,7 @@
 logging.basicConfig(level=logging.DEBUG)
 
 SIMPLE = '[28/Oct/2015:15:18:14 +0000] GET /a/uth-rhd/api/case/attachment/a26f2e21-5f24-48b6-b283-200a21f79bb6/VH016899R9_000839_20150922T034026.MP4 HTTP/1.1 401 0.242'
+SIMPLE_TZ = '[28/Oct/2015:15:18:14 +0530] GET /a/uth-rhd/api/case/attachment/a26f2e21-5f24-48b6-b283-200a21f79bb6/VH016899R9_000839_20150922T034026.MP4 HTTP/1.1 401 0.242'
```

Comment:
it worries me that the servers are not in UTC. i feel like we make that assumption in many places